lib/cartopy/mpl/geoaxes.py

```diff
@@ -988,22 +989,38 @@ def stock_img(self, name='ne_shaded'):
         """
         Add a standard image to the map.
 
-        Currently, the only (and default) option is a downsampled version of
-        the Natural Earth shaded relief raster.
+        Currently, there are 2 options:
 
+        1. 'ne_shaded'(default) a downsampled version of the Natural Earth
+           shaded relief raster.
+           https://github.com/SciTools/cartopy/blob/master/lib/cartopy/data/raster/natural_earth/50-natural-earth-1-downsampled.png
+        2. 'etopo' a downsampled version of global relief model of Earth's
+           surface that integrates land topography and ocean bathymetry. This
+           option is the same as the etopo from Basemap.
+           https://www.ngdc.noaa.gov/mgg/image/color_etopo1_ice_low.jpg
         """
+        if name not in ['ne_shaded', 'etopo']:
+            raise ValueError('Unknown stock image %r.' % name)
+
         if name == 'ne_shaded':
             import os
             source_proj = ccrs.PlateCarree()
             fname = os.path.join(config["repo_data_dir"],
                                  'raster', 'natural_earth',
                                  '50-natural-earth-1-downsampled.png')
+        elif name == 'etopo':
+            import os
+            source_proj = ccrs.PlateCarree()
```

Comment:
Move these outside of the if/elif clause because they are the same for both cases?